tests/integration/test_client.py

```diff
@@ -102,6 +103,40 @@ def test_upload_get_delete_artifact(client, tmpdir):
         assert_response(resp)
 
 
+@pytest.mark.parametrize('client', [{'jsonify': True}], indirect=True)
+def test_delete_many_artifacts(client: Client):
```

Comment:
```suggestion
def test_delete_artifacts(client: Client):
```

Reply:
I prefer long version. `delete_artifact` and `delete_artifacts` are too same.